Orange/widgets/evaluate/owrocanalysis.py

```diff
@@ -448,8 +448,7 @@ def _initialize(self, results):
             names = ["#{}".format(i + 1)
                      for i in range(len(results.predicted))]
 
-        self.colors = colorpalette.ColorPaletteGenerator(
-            len(names), colorbrewer.colorSchemes["qualitative"]["Dark2"])
+        self.colors = colorpalette.ColorPaletteGenerator(len(names))
```

Comment:
The palette that we had before could have been chosen for a reason. It should be used if possible; switch to a more generic palette only if needed.